docker/user_python-ubuntu.sh

```diff
@@ -26,16 +26,14 @@ fi
 # Python
 case "$PYTHON" in
 miniconda)
-  #curl -L https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh > miniconda.sh
-  curl -L https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh > miniconda.sh
+  curl -L https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh > miniconda.sh
```

Comment:
seems like build fails because system libboost needs py3.10, not 3.11 :/

```suggestion
  curl -L https://repo.anaconda.com/miniconda/Miniconda3-py310_23.5.2-0-Linux-x86_64.sh > miniconda.sh
```